src/pygef/cpt.py

```diff
@@ -140,6 +141,7 @@ class CPTData:
     delivered_vertical_position_datum: VerticalDatumClass
     delivered_vertical_position_reference_point: str
     column_void_mapping: dict | None
+    headers: dict | None
```

Comment:
```suggestion
    headers: dict
```

I think it's best to always create a map for the headers, and in other formats which don't support them just create an empty one.